cmake/FindLibintl.cmake

```diff
@@ -18,9 +18,8 @@ find_path(LIBINTL_INCLUDE_DIR
     PATH_SUFFIXES gettext
 )
 
-find_library(LIBINTL_LIBRARY
-    NAMES intl libintl
-)
+find_library(LIBINTL_LIBRARY NAMES libintl${CMAKE_STATIC_LIBRARY_SUFFIX})
```

Comment:
Just write `libintl.a`, this won't work on windows anyways due to the `lib` prefix.

Reply:
TODO: ayo hold up. Where does the static libintl even come from? I see we don't even build it on non-windows.